js/models/employee.ts

```diff
@@ -8,6 +8,14 @@ export const Employee = z.object({
   badge_serials: z.array(z.string()),
 });
 
+export const fullName = (emp: Employee | null) => {
```

Comment:
**nitpick:** could this be named something like `displayName` or `fullDisplayName` instead? Usually when I think of "full name" I think of it as including first, middle, and last (with all the usual caveats about names being complicated and not everyone having a name of that form to begin with).

Reply:
c959b6e